shared/output/gapic/templates/showcase/lib/google/showcase/v1beta1/messaging/paths.rb

```diff
@@ -31,39 +31,84 @@ module V1beta1
       module Messaging
         # Path helper methods for the Messaging API.
         module Paths
+          ##
+          # Create a fully-qualified  resource string.
+          #
+          # The resource will be in the following format:
+          #
+          # `projects/{project}`
+          #
+          # @param project [String]
+          #
+          # @return [String]
+          def _path project:
```

Comment:
??

Reply:
This is because Showcase defines a resource without providing a valid `resource_type` in the annotation. A valid value is required per https://aip.dev/client-libraries/4231, so this looks to be a case where the showcase protos are incorrect.

Reply:
Implementation is updated. Please review.